test/test_cpp_extensions.py

```diff
@@ -15,7 +15,7 @@
     import torch_test_cpp_extension.cpp as cpp_extension
     import torch_test_cpp_extension.msnpu as msnpu_extension
 except ImportError:
-    warnings.warn(
+    raise RuntimeError(
```

Comment:
Running this file with just a warning the build continued, generated quite a bit of output, and eventually ended with an error. The warning at the beginning of the output wasn't obvious to me.